### ISSUE TITLE
When calculating mAP on the test set of the DOTA dataset, it prompts that there are no labels.

### ISSUE THREAD

**Issue by a5336033** [ultralytics/ultralytics#10169]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Hello, I noticed that your official website evaluates the performance of DOTAv1.0 on OBB using the test set. However, when I try to evaluate the model using the test set, it prompts that there are no labels available. I am able to evaluate the model using the validation set without any issues. The dataset was downloaded from your official website.
![tmp1677](https://github.com/ultralytics/ultralytics/assets/61953259/025f3992-059c-4e69-9160-ae19a3e618ce)

### Additional

Does OBB training provide pre-trained models, or does the website only offer models that have been fully trained?

**Comment by glenn-jocher** [comment]
Hello! 👋 It seems like you've encountered a common issue where the test set might not have labels included, which is typical for many datasets to ensure unbiased evaluation. For the DOTA dataset's test set, labels are indeed not publicly available; they are usually held back by dataset providers for competition purposes or to enable unbiased benchmarking.

For evaluating model performance with a set that lacks labels, you could use predictions made on the validation set or consider submitting your predictions to the official DOTA evaluation server if they provide one. This way, the server can compare your predictions against the withheld test labels and give you the performance metrics.

Regarding pre-trained models for OBB (Oriented Bounding Box) training on the DOTAv1.0 dataset, if provided on our website, these models have been fully trained. You can use them as a strong starting point for further training or for inference directly.

If you're looking to evaluate on the validation set (which has labels) and have done so successfully, here's a quick reminder of the command structure, just in case:

```bash
yolo val model=<your_model>.pt data=<your_data>.yaml
```

Replace `<your_model>` and `<your_data>` with the paths to your model and data configuration file, respectively.

For any further assistance or more specific queries, feel free to ask. We're here to help! 😊
